week05/my-todolist/src/App.js

```diff
@@ -1,32 +1,42 @@
-import { useEffect, useState } from "react";
-import Header from "./Header";
-import TodoList from "./TodoList";
-import TodoAdd from "./TodoAdd";
-import StateTest from "./StateTest";
+import { Routes, Route, NavLink } from "react-router-dom";
+//import StateTest from "./StateTest";
 import "./App.css";
-
-const TODO_LIST_KEY = "todoList";
+import MyTodolist from "./components/pages/MyTodolist";
+import MyPlaylist from "./components/pages/MyPlaylist";
 
```

Comment:
1. `IMG_4976.jpg` 파일을 src 폴더 아래로 옮깁니다.
2. 이미지를 불러옵니다.
3. `useEffect`, `useState` import합니다.

```suggestion
import { useEffect, useState } from "react";
import bgTodolist from "./back.jpg";
import bgPlaylist from "./IMG_4976.jpg";
```